.github/workflows/azure-static-webapp.yml

```
@@ -0,0 +1,79 @@
+name: Azure Static Web Apps CI/CD
+
+on:
+  push:
+    branches:
+      - master
+      - release/*
+  pull_request:
+    types: [opened, synchronize, reopened, closed]
+    branches:
+      - master
+      - release/*
+
+jobs:
+  build_and_deploy_job:
+    env:
+      DIST_PATH: Uno.Gallery/Uno.Gallery.Wasm/bin/Release/net5.0/dist
```

```suggestion
      DIST_PATH: Uno.Gallery/Uno.Gallery.Wasm/bin/Release/net5/dist
```

Because the targetframework is `net5`